external/netstandard/netstandard.depproj

```diff
@@ -6,14 +6,21 @@
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
     <OutputPath>$(RefRootPath)/$(TargetGroup)</OutputPath>
     <ProjectJsonTemplate Condition="$(NuGetTargetMoniker.Contains('Version=v1.'))">$(MSBuildProjectDirectory)\netstandard1.x\project.json.template</ProjectJsonTemplate>
+    <_NETStandardTFMFolder Condition="'$(NuGetTargetMoniker)' == '.NETStandard,Version=v2.0'">netstandard2.0</_NETStandardTFMFolder>
+    <_NETStandardTFMFolder Condition="$(NuGetTargetMoniker.StartsWith('.NETFramework'))">net461</_NETStandardTFMFolder>
```

Comment:
I'm not filtering yet.  Right now I am just overwriting them but was planning to filter.  I can continue to overwrite if you think its ok.